README.md

```diff
@@ -64,6 +64,9 @@ Savvy Widget supports a number of Javascript callbacks that you can use for anal
       // OPTIONAL: Set this value to true if you would like to skip the intermediary login interview questions
       directLogin: false,
 
+      // If the Savvy Widget is going to be embedded in a native web view, set isWebView: true
```

Comment:
Since this is a nullable field, I think we should include the OPTIONAL: portion in the documentation.